experimental/generation/runbot/RunBot.csproj

```
@@ -30,9 +30,10 @@
     <PackageReference Include="Jint" Version="2.11.58" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.12.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.9.0-preview-200403-117546" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Debugging" Version="4.8.0-rc0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.8.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.0-preview-200504-127776" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.9.0-previewB-200501-127440" />
```

@chrimc62 this build was temporary 4.9.0-previewB-200501-127440, and has been removed due to a myget sorting issue

```suggestion
    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.9.0-rc1" />
```